torch/_inductor/compile_fx.py

```diff
@@ -619,6 +522,92 @@ def compile_fx_inner(
 
     fx_graph_remote_cache = should_use_remote_fx_graph_cache()
     inputs_to_check = get_input_idxs_to_check(example_inputs, static_input_idxs)
+
+    def codegen_and_compile(
+        gm,
+        example_inputs,
+        inputs_to_check,
+        fx_kwargs,
+    ):
+        """
+        This function calls fx_codegen_and_compile and also adds some extra metadata to the resulting
+        compiled fx graph. The metadata is saved to FXGraphCache.
+        """
+        compiled_graph = fx_codegen_and_compile(gm, example_inputs, **fx_kwargs)
+        if isinstance(compiled_graph, str):
+            # No need to do any postprocessing if we return a string
```

Comment:
When does that happen? (just out of curiosity)

Reply:
nvm, answered below in a comment

Reply:
yea, why is this ? comment inline might be useful

Reply:
Will add an inline comment: we only return a string in aot mode, which doesn't run any of the post-processing steps